### DISCUSSION TITLE
Cannot connect to https://app.planetscale.com

### DISCUSSION THREAD

**Discussion by Guangsudalao** [planetscale/discussion#249] [Bug Reports] [answered]
When I was https://planetscale.com/ Trying to connect to https://app.planetscale.com/ When trying to log in, the server returns 500 errors. What should I do?
![Screenshot_20220810_170623_com android chrome](https://user-images.githubusercontent.com/73008998/183862086-3a633388-2db1-468b-bcdf-b21ddbee4a92.png)
Sometimes the server returns a 404 errors.
![183865531-1c7c9e39-2887-459f-9807-fce4297889a8_edit_29277004879386](https://user-images.githubusercontent.com/73008998/183868637-f03f9113-3839-4cd0-9f3f-f7cc86f1ba14.png)

**Comment by fooforge** [comment, marked as answer]
@Guangsudalao Looks like you may have hit a bug.

Would you please create a ticket at https://support.planetscale.com/ so that we can look into it further? When you register an account for the support portal, please make sure you use the same email address as your PlanetScale account or let us know your account name in the ticket so that we know where to look. Thank you!

**Reply by Guangsudalao** [reply]
OK,I will open a ticket on https://support.planetscale.com/
